docs/overview.md

```diff
@@ -0,0 +1,44 @@
+# KILT Protocol
+
+KILT is a decentralized system to own and manage your identity. 
+It's a simple protocol for creating, claiming, issuing, presenting and verifying digital credentials attached to it. 
+In contrast to other centralised or peer-to-peer solutions for this, KILT features self-sovereign data as well as revocable credentials using blockchain technology.
+KILT was built to be a business enabler, not only for the software industry, but also for any entity, which has or wishes to establish a business model based on trust. 
+We believe KILT will be an essential building block of the Web 3.0. 
+
+In particular KILT provides:
+
+* A universal blockchain protocol for individuals, organisations, objects, and artificial intelligences to claim arbitrary attributes about themselves and get them attested by trusted entities.
+* Mechanisms for putting claim holders in control of their data by storing the information on their storage and by giving them the choice which information they want to disclose to whom.
+* A Trust Market for the Attesters of such claims, which allows trusted entities to attach prices to their valuable attestation work and get paid.
+
+Our main goal with KILT is to generate a level playing field for companies to explore new business models related to trust relationships and data sovereignty.
+With KILT we enable businesses and governments to rely on a common standard which is owned by everyone participating and not by a single company.
+
+## The problem
+
+For hundreds of years, identity and trust between entities has been organized in a decentralised way. 
+For example official, personal documents like passports were issued by trusted entities and handed out to the holders. 
+Those holders were then under full control of their credentials and could use them where ever they wanted. 
```

Comment:
I see your point, but the passport is more an analogy for the credential not the DID. There is no real equivalent to a DID in the real world that I can actually think of. Identity only really exists as a concept and its indeed build up by all the credentials and traits (personal observations) associated with this identity.

Also the authorities are kind of decentralized. Just think about all the different Einwohnermeldeämter, or the different drivers license departments in the US.

Reply:
Only because they are many doesn't mean they are decentralized.

Reply:
Also what a DID does is actually giving you an identifier. In Germany you don't get that with your ID Card because for historic reasons numbers are no more assigned to people. But other countries to that. They assign identifiers to their citizens. And this is a central authority. You have no control over your identifier.
The DID is different. You don't get it assigned you take it yourself and you manage it by yourself. That's the difference.

Reply:
@weichweich I made my statement about how we historically had a decentralised system much weaker. I still think that the actual narrative of "trust and attestations were decentralized, then got centralized in web 2.0 and now we are decentralizing again" still valid and a good point to make. I totally agree that having a DID is superior to having an id assigned by somebody, but that's not what I want to convey in this paragraph.

Do you have any suggestions how to improve from this point? I thought really long about this, but except deleting it and making up something completely new I don't see how to adjust it (except that I made the wording a bit weaker)

Reply:
In my opinion I would rewrite it. Also I would not write `trust between entities` but `trust between people`. I think the scope of `trust has been decentralized` is just to big to be true. For me it sounds like before web2 the world was perfect. Everything was decentralised. Which is clearly not the case.

Reply:
@weichweich I rewrote it and made clear that we were already on our way to centralisation when the first passport was given out. I hope this is good now, I think its v3 or v4 already :-P

Reply:
OMG That's really good. I like it!

Thanks for being patient with me and my complaints. 😓

Reply:
I would change the first sentence to: For hundreds of years, trust between identities has been organized in a decentralised way.